lua/location_utils.lua

```diff
@@ -28,7 +28,11 @@ M.telescope_list_or_jump = function(title, params, locations, lsp_client, opts)
       end
     end
 
-    vim.lsp.util.jump_to_location(locations[1], lsp_client.offset_encoding, opts.reuse_win)
+    if vim.lsp.util.show_document ~= nil then
+      vim.lsp.util.show_document(locations[1], lsp_client.offset_encoding, { reuse_win = opts.reuse_win })
```

Comment:
should this map also have `focus = true`, looking at https://neovim.io/doc/user/deprecated.html#vim.lsp.util.jump_to_location that's the suggested refactoring, not exactly sure if default value (`focus=true`) is used if map arg is passed in 🤔

Reply:
@Hoffs 

When the arguments table is only `{ reuse_win = opts.reuse_win }`, the default `focus = true` is still being used.

Without specifying `focus` (or with `focus = true`) the new location is opened in a new buffer that gets focused.
With `focus = false` a new buffer and split will be opened with that location, and the cursor stays in the same place, it doesn't jump to the newly opened location.

I'd prefer not specifying focus as we don't need to change the default behaviour.